pkg/util/log/log.go

```diff
@@ -392,9 +384,6 @@ func (sw *loggerPointer) criticalf(format string, params ...interface{}) error {
 
 // getLogLevel returns the current log level
 func (sw *DatadogLogger) getLogLevel() seelog.LogLevel {
-	sw.l.RLock()
```

Comment:
Same: do we need a comment for this function ?

Reply:
Not really the name is pretty clear with what it does I'll remove the comment in this PR, thank you